metadata/ComputerVision__2304105__metadata.json

```diff
@@ -19,8 +19,8 @@
   "tenantName": "UiPath",
   "minAIFabricVersion": "22.10",
   "languageVersion": 3,
-  "version": 7,
-  "customVersion": "23.10.1",
-  "imagePath": "cv2310:23.10.5.2",
+  "version": 2304105,
```

Comment:
This version is not displayed anywhere, why this change?

Reply:
When present, the custom version is displayed.

Reply:
We want to change it to represent the CV Model version, so in the future it will be harder to put a number there presented in another branch in order to break something. Since this number is unique per cv release, it will also be unique. Does it sounds good?

Reply:
It sounds good but we need to check somehow that this is working.

I wouldn't do this change in stable.


Maybe next time, in testing, ane merge it in stable after testing.

Reply:
Ok, I've reverted last changes

Reply:
BTW, instead of `2304105` it should be either `23041005` or `230406`

Reply:
@MaxTranced maybe is working but we I would like to test in testing first before going to stable.